styles/alphanumeric.csl

```diff
@@ -22,8 +22,11 @@
       <key variable="issued" />
     </sort>
     <layout prefix="[" suffix="]" delimiter=", ">
-      <text variable="citation-label" />
-      <text variable="year-suffix" />
+      <group delimiter=", ">
+        <text variable="citation-label"/>
+        <text variable="year-suffix"/>
+        <text variable="locator" prefix=" " />
```

Comment:
Is there a reason for the nested group with the same delimiter and this prefix? What is your intended output?

Of note, `@citation[abc]`  produced `[Cit25a, abc]` in Typst 0.8.0, before the regression. We should probably stick to that.